apps/studio/components/ui/AIAssistantPanel/Message.tsx

```diff
@@ -21,10 +25,9 @@ const baseMarkdownComponents: Partial<Components> = {
   a: Link,
 }
 
-interface MessageProps {
+interface MessageProps extends Omit<VercelMessage, 'id'> {
   id: string
-  role: 'function' | 'system' | 'user' | 'assistant' | 'data' | 'tool'
-  content?: string
+  message: VercelMessage
```

Comment:
Should this be `Message` from `ai/react` or from `@ai-sdk/react`?

This is causing the type errors and I'm not sure how to deconflict here - `AIAssistant`'s `chatMessages` from `useChat` returns messages as typed from `ai/react`, but `Messsage.tsx` expects it to be typed from `@ai-sdk/react`